command-utility-tool/src/main/java/com/utility/InputHandler.java

```diff
@@ -0,0 +1,108 @@
+package com.utility;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+/**
+ * This class handles user input and retry 3 times if input is invalid.
+ * If input is valid, then check if the input path and file name is valid
+ * If input bye, then exit the process
```

Comment:
I think "bye" is no longer valid to exit the code?

Reply:
yes, will modify the description